lua/cinema/weapons/weapon_spraypaint/shared.lua

```diff
@@ -67,6 +67,7 @@ end
 
 function SWEP:PrimaryAttack()
     local ply = self:GetOwner()
+    if not (IsValid(ply) and ply:Alive()) then return end
```

Comment:
is this to fix an error on server or client? it seems like the weapon shouldn't be able to primary attack unless it has a living owner, unless maybe the localplayer is invalid on client somehow

Reply:
an error is thrown when a person dies while using the spraypaint

Reply:
Oh I see why. The check should be moved down to line 117-ish i think